core/src/vote_simulator.rs

```diff
@@ -65,7 +65,6 @@ impl VoteSimulator {
         }
     }
 
-    #[cfg(feature = "dev-context-only-utils")]
```

Comment:
Why is it redundant?

Reply:
On top of the file, the whole module is decorated with "dev-context-only-utils".